packages/lesmis/src/components/DashboardReportModal.js

```diff
@@ -161,27 +150,17 @@ export const DashboardReportModal = ({
                   isLoading={isLoading}
                   startDate={startDate}
                   endDate={endDate}
-                  granularity={viewContent?.granularity}
+                  granularity={periodGranularity}
```

Comment:
The period granularity as part of the dashboard response and not in the report response so use `periodGranularity` from the dashboard item list which was fetched upstream. This fixes bug [3012](https://github.com/beyondessential/tupaia-backlog/issues/3012)